tests/TestCase.php

```diff
@@ -21,16 +23,23 @@ abstract class TestCase extends BaseTestCase
 
     const DEFAULT_KEY = 'masterKey';
```

Comment:
Consider adding a visibility for this const 🙂

Reply:
Nice, added.